<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AloneSetting</class>
 <widget class="QMainWindow" name="AloneSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBoxPlayers">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Количество играков</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMoney">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Начальная сумма</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string> Small Blind</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBlind">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>110</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="game">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnMainMenu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>В главное меню</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBUtton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBUtton</sender>
   <signal>clicked()</signal>
   <receiver>AloneSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
